### PR TITLE
Common meters

### PR DESCRIPTION
**Pull request by Timer00**
I am creating this PR for the common meters to optimize the conflict resolution once individual meters are approved.

### CODE REVIEW

File: symbols/ac_voltmeter_vert.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { rotateSymbol } from "drawing/rotateSymbol"
+import dc_voltmeter_horz from "./ac_voltmeter_horz"
+import { Primitive } from "drawing/types"
+
+console.log(dc_voltmeter_horz.primitives)
```
Comment (seveibar) [inline symbols/ac_voltmeter_vert.ts:5]:
```suggestion
```

**Comment by seveibar** [comment]
looking good so far, i guess tag me for review when it's ready? I would merge 1 by 1 personally, maybe we can figure out how to resolve the conflicts automatically somehow, e.g. moving the common generated files into a build step and not committing them

**Comment by Timer00** [comment]
> looking good so far, i guess tag me for review when it's ready? I would merge 1 by 1 personally, maybe we can figure out how to resolve the conflicts automatically somehow, e.g. moving the common generated files into a build step and not committing them

100% agreed. It looks like this has been done already but running bun dev runs build. Which creates these file changes necessary for rendering the code (I believe). So just gotta be careful about not committing those.

File: symbols/ac_voltmeter_vert.ts (hunk 1 above)
Comment (Timer00) [reply symbols/ac_voltmeter_vert.ts:5]: Addressed this here: #94

**Comment by Timer00** [comment]
Found different solution by simply not including the frequently modified files in PR since they are modified on build anyways.

**Pull request closed without merging**